cost-analyzer/values.yaml

```diff
@@ -546,6 +546,12 @@ serviceAccount:
   annotations: {}
   # name: kc-test
 
+# Define global data filters for specific user groups
+# Disabled by default
+groupFilters:
+  enabled: false
```

Comment:
Do we need this if the fields are empty? Just feels like more for users to manage...

Reply:
^ That's a good point